src/main/java/gov/usds/case_issues/services/KPIService.java

```diff
@@ -0,0 +1,84 @@
+package gov.usds.case_issues.services;
+
+import java.time.DayOfWeek;
+import java.time.ZonedDateTime;
+import java.time.temporal.TemporalAdjuster;
+import java.time.temporal.TemporalAdjusters;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.validation.annotation.Validated;
+
+import gov.usds.case_issues.db.repositories.BulkCaseRepository;
+import gov.usds.case_issues.services.model.CaseGroupInfo;
+import gov.usds.case_issues.validators.TagFragment;
+
+@Service
+@Validated
+public class KPIService {
+	private static final int WEEKS = 10;
+	private static TemporalAdjuster START_OF_WEEK = TemporalAdjusters.previousOrSame(DayOfWeek.MONDAY);
+
+	@Autowired
+	private BulkCaseRepository _bulkRepo;
+	@Autowired
+	private CaseListService _caseListService;
+
+	public Map<String, Object> getKPIData(
+		@TagFragment String caseManagementSystemTag,
+		@TagFragment String caseTypeTag
+	) {
+		CaseGroupInfo translated = _caseListService.translatePath(
+			caseManagementSystemTag,
+			caseTypeTag
+		);
+		HashMap<String, Object> kpis = new HashMap<String, Object>();
+		Long caseManagementSystemId = translated.getCaseManagementSystemId();
+		Long caseTypeId = translated.getCaseTypeId();
+		kpis.put("ResolvedTickets", getResolvedTickets(caseManagementSystemId, caseTypeId));
```

Comment:
Nitpickiest of nitpicks: camel case object keys should start with a lower case letter. (No, I don’t think this is a thing that is worth changing right now.)